resources/views/cookies.blade.php

```diff
@@ -67,11 +67,17 @@
     </div>
 </aside>
 
-{{-- STYLES & SCRIPT : feel free to remove them and add your own --}}
-
-<script data-cookie-consent>
-    {!! file_get_contents(LCC_ROOT . '/dist/script.js') !!}
-</script>
+{{-- STYLES & SCRIPT : feel free to remove them and add your own
+    This script can't be removed --}}
+@if(!$isReset)
+    <script data-cookie-consent>
+        let element = document.querySelector('#cookies-policy');
+        if (element) {
+            element.classList.add('cookies--pre-init');
+            element.classList.add('cookies--closing');
+        }
+    </script>
+@endif
```

Comment:
Don't like this at all. Should be handled inside the `modal.js` init method. In all cases, the `#cookies-policy` element should be hidden when JS is enabled and the modal's script has not been initiated.